docs/ff-concepts/notifications/notifications.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: /concepts/notifications
+title: Notifications
+description: Learn how to add notifications in FlutterFlow.
+tags: [Notifications, Concepts]
+sidebar_position: 0
+keywords: [FlutterFlow, Notifications, Concepts]
+---
+
+# Notifications
+
+**Mobile Notifications** are alerts or messages that appear on a user's device outside the normal UI flow of an app. They can inform the user of time-sensitive or high-priority messages, events, or actions that require attention. Notifications may appear as banners, alerts, pop-ups, or lock-screen notifications, depending on user preferences and platform design guidelines.
```

Comment:
Why are we just calling it mobile? There can be web notifications as well right?